src/main/resources/view/DarkTheme.css

```diff
@@ -350,3 +350,30 @@
     -fx-background-radius: 2;
     -fx-font-size: 11;
 }
+
```

Comment:
it may be useful to have a new css file for this instead of throwing it in dark theme

Reply:
Ok, let's have a look at this at v1.6